### DISCUSSION TITLE
Requiring all conversations to be resolved

### DISCUSSION THREAD

**Discussion by andrewdang17** [community/community#10304] [Pull Requests] [answered]
I would love a feature to require all PR comments to be resolved (by either the author or reviewers) before being able to merge or re-requesting a review from someone again.

**Comment by jakeleventhal** [comment]
This feature exists now

**Reply by mroy-seedbox** [reply]
Indeed! Under branch protection rules: 
![image](https://user-images.githubusercontent.com/66686238/168399626-22573378-b3ac-47c0-8be1-eafb309ce2e8.png)

**Reply by wolfch-elsevier** [reply]
How can I circumvent this?  Neither the PR submitter nor the reviewer can unblock this.

**Reply by mroy-seedbox** [reply]
@wolfch-elsevier: There's another setting to prevent anyone from circumventing this:
![image](https://user-images.githubusercontent.com/66686238/235239326-178fa952-80cb-4d3c-9bf2-de189802182d.png)

Otherwise, you can just go through the conversations/comments and resolve them all.

But generally, the goal of branch protection rules is specifically that _**they cannot be circumvented**_.

**Reply by bernardoVale** [reply]
Looks like the option no longer exists.

**Reply by mroy-seedbox** [reply]
It's still there:
![image](https://github.com/user-attachments/assets/1eb89c34-bda2-4ae3-943a-4b38de2f9a66)

**Reply by raven-wing** [reply, marked as answer]
It's just hidden very well:
` Rulesets-> Require a pull request before merging -> Show additional settings -> Require conversation resolution before merging`

**Reply by bernardoVale** [reply]
indeed @raven-wing 🤣
